src/view/pointPath.js

```diff
@@ -1,29 +1,44 @@
-export const createPointPathTemplate = () => (
-  `<li class="trip-events__item">
+import dayjs from 'dayjs';
+
+export const createPointPathTemplate = (point) => {
+
+  const offers = point.offers.offers;
+  const type = point.type;
+  const destination = point.destination;
+  const price = point.basePrice;
+  const dateToHours = dayjs(point.dateTo).format('hh:m');
+  const dateFromDay = dayjs(point.dateFrom).format('MMM D');
+  const dateFromHours = dayjs(point.dateFrom).format('hh:m');
+  const templateOffers = (obj) => (`<li class="event__offer">
+          <span class="event__offer-title">${obj.title}</span>
+          &plus;&euro;&nbsp;
+          <span class="event__offer-price">${obj.price}</span>
+        </li>`
+  );
+
+  const renderOffers = (arrOffer) => arrOffer.map((item) => templateOffers(item)).join('');
```

Comment:
и это тебе тоже тут не нужно, это чистая функция, что получает конкретный набор данных, и возвращает конкретный результат. 

Также название arrOffer поменяй на offers это понятное и доступное название. item лучше не используй, это же offer 

ну и в мэп зачем передавать новый коллбек, который просто передает и вызывает другую функцию? 

```suggestion
  const renderOffers = (arrOffer) => arrOffer.map(templateOffers).join('');
```

повтори понятие колбеков